test/e2e/config/helm.yaml

```diff
@@ -3,31 +3,31 @@ managementClusterName: caaph-e2e
 images:
   - name: ${MANAGER_IMAGE}
     loadBehavior: mustLoad
-  - name: registry.k8s.io/cluster-api/cluster-api-controller:v1.7.4
```

Comment:
Apparently we forgot to bump the e2e test versions of CAPI in previous update PRs. 😮